src/libraries/Microsoft.NETCore.Platforms/src/runtimeGroups.props

```diff
@@ -17,7 +17,7 @@
     <RuntimeGroup Include="alpine">
       <Parent>linux-musl</Parent>
       <Architectures>x64;x86;arm;armv6;arm64;s390x;ppc64le</Architectures>
-      <Versions>3.6;3.7;3.8;3.9;3.10;3.11;3.12;3.13;3.14;3.15;3.16;3.17</Versions>
+      <Versions>3.14;3.15;3.16;3.17</Versions>
```

Comment:
Alpine 3.13 and previous are EOL 2022-11-01. (https://www.alpinelinux.org/releases/)